pages/advanced/dvb-font-downloading.md

```diff
@@ -0,0 +1,135 @@
+```
+
+The table below details the attributes themselves.
+
+| Name             | Type   | Description                                                                            |
+|:---------------- |:-------|:---------------------------------------------------------------------------------------|
+| `dvb:url`        | URI    | The URL of the font to download. Can be absolute or can make use of relative BaseURLs. |
+| `dvb:fontFamily` | String | The font family used in EBU-TT-D documents.                                            |
+| `dvb:mimeType`   | String | The mimeType of the font available from the URL.                                       |
+
+### Property Descriptor Example
+
+```xml
+<SupplementalProperty
+    schemeIdUri="urn:dvb:dash:fontdownload:2014"
+    value="1"
+    dvb:url="https://example.com/fonts/SubtitleDisplay.woff"
+    dvb:fontFamily="SubtitleDisplay"
+    dvb:mimeType="application/font-woff"
+/>
+```
+
+### Mimetype Support
+
+The DVB DASH specification denotes support for two mimeTypes:
+
+* `application/font-sfnt` - This covers `.ttf` and `.otf` fonts.
+* `application/font-woff` - Covering `.woff` fonts.
+
+No further support for unspecified mimeTypes has been provided.
+
+### Supplemental vs Essential Property Descriptors
+
+SupplementalProperty descriptors are used when it is acceptable to show the subtitles in another font if the download fails.
+EssentialProperty descriptors are used when the subtitles must not be shown at all if the download fails.
+
+As such the player acts based on what kind of property descriptor was used to describe the downloadable font.
+If a `<SupplementalProperty>` descriptor was used and download fails, then the `<AdaptationSet>` containing the descriptor continues to be presented as if the `<SupplementalProperty>` descriptor was not present.
+
+If an `<EssentialProperty>` descriptor was used and download fails, then the `<AdaptationSet>` containing the descriptor is not be presented at all.
+
+What this looks like on a client is described in a later section.
+
+## Using the Downloaded Fonts
+
+### TTML Font Family Attribute
+
+The EBU-TT-D subtitles need to indicate that they want to use the downloaded font.
+To do this, they must use the same `fontFamily` name as described by the `dvb:fontFamily` attribute in the MPD.
+
+So, for example, if we have this attribute in an MPD,
+
+```xml
+<SupplementalProperty
+    dvb:fontFamily="SubtitleDisplay"
+/>
+```
+
+we would need to ensure this is present in the subtitle TTML documents to put the font to use.
+
+```xml
+<style
+    tts:fontFamily="SubtitleDisplay, Arial, default"
+/>
+```
+
+Note that fallback fonts can still be specified.
+In this case we have "Arial" specified as an expected system font, and "default" which is interpreted by [imsc](https://www.w3.org/TR/ttml-imsc1.1/) as the default monospace serif font provided by the browser/device in use.
```

Comment:
I think it's fine to use IMSC 1.0.1 for the reference here - any reason for choosing 1.1 in particular?

Reply:
No reason other than higher numbers!

Reply:
Have changed in text.

Reply:
Thanks. The reason I think it's better here to use lower version numbers is because that widens the overall compatibility set downstream, in the sense that there are no features needed for using fonts that are not present in the earliest version of IMSC. Later versions add features that aren't particularly relevant.